### ISSUE TITLE
AccordProject documenation contains many broken links, grammatical and punctuation errors

### ISSUE THREAD

**Issue by prakharsr** [accordproject/techdocs#336]
AccordProject documenation contains many broken links, grammatical and punctuation errors. I will mention those issues that I was able to catch and fix while reviewing the documentation:

## Broken, incorrect links - 

1. **fix incorrect link (markup-template)->(markup-preliminaries)** affecting 4 files (docs/accordproject-slc.md, docs/accordproject-template.md, website/versioned_docs/version-0.21/accordproject-slc.md, website/versioned_docs/version-0.21/accordproject-template.md) at url https://docs.accordproject.org/docs/accordproject-slc.html, fixed in 4d4548d455cacf55cb49caaac369859c3a7a6bc6
2. **version (0.12, 0.20) - fix broken and incorrect links, correct missing late10b.png and add late17.png at the end** affecting 2 files (website/versioned_docs/version-0.12/advanced-latedelivery.md, website/versioned_docs/version-0.20/tutorial-latedelivery.md) at url https://docs.accordproject.org/docs/0.12/advanced-latedelivery.html, fixed in 08809520548c3dc924ddd3f7dc1dc3a1a83ace3a
3. **fix broken link (ref-decorators) -> (ref-concerto-decorators)** affecting 2 files (docs/model-decorators.md, website/versioned_docs/version-0.21/model-decorators.md) at url https://docs.accordproject.org/docs/model-decorators.html, fixed in 00fc8a75f05c40c13b6ef1a676e78fbfc7e6c257
4. **fix broken link (accordproject-specification) -> (accordproject-template)** affecting 4 files (docs/logic-ergo.md, website/versioned_docs/version-0.12/accordproject.md, website/versioned_docs/version-0.12/logic-ergo.md, website/versioned_docs/version-0.20/logic-ergo.md) at url https://docs.accordproject.org/docs/logic-ergo.html, fixed in 5bbb349e98f01f46ac10a653eb23ecc32afff85c
5. **fix broken link (ergo-reference.html#condition-expressions) -> (ref-logic#condition-expressions)** affecting 3 files (docs/logic-simple-expr.md, website/versioned_docs/version-0.12/logic-simple-expr.md, website/versioned_docs/version-0.20/logic-simple-expr.md) at url https://docs.accordproject.org/docs/logic-simple-expr.html, fixed in d7407045257f451f5b0d03415c1b92b8c3cd3b0f
6. **fix hardcoded docs link (https://docs.accordproject.org/docs/cicero-api.html) to (ref-cicero-api in v0.21 docs) and (cicero-api in v0.20 docs)** affecting 2 files (docs/tutorial-nodejs.md, website/versioned_docs/version-0.20/tutorial-nodejs.md) at url https://docs.accordproject.org/docs/tutorial-nodejs.html, fixed in 369efd4dd3f7b8ece7b84fc31529934bb94a42ce
7. **fix broken link (markdown-templatemark) -> (markup-templatemark)** affecting 2 files (docs/ref-markus-cli.md, website/versioned_docs/version-0.21/ref-markus-cli.md) at url https://docs.accordproject.org/docs/ref-markus-cli.html, fixed in a7552a08fb0571ec7c4674562a0b4f55a0754895
8. **fix broken link (0.13to0.20.html)->(ref-migrate-0.13-0.20.html)** affecting 3 files (docs/ref-ergo-cli.md, website/versioned_docs/version-0.20/ergo-cli.md, website/versioned_docs/version-0.21/ref-ergo-cli.md) at url https://docs.accordproject.org/docs/ref-ergo-cli.html, fixed in 407440bcd862d118ad17a1d71afe58e62d49fb2e

## Grammatical and punctuation errors -

1. **fix case  ("Smart COntracts" -> "Smart Contracts")** affecting 2 files (docs/accordproject-faq.md, website/versioned_docs/version-0.21/accordproject-faq.md) at url https://docs.accordproject.org/docs/accordproject-faq.html, fixed in 62e378d85d12bd2c998b0400a3139fc5b950818b
2. **fix punctuation ("To do so you," -> "To do so, you")** affecting 3 files (docs/started-hello.md, website/versioned_docs/version-0.20/started-hello.md, website/versioned_docs/version-0.21/started-hello.md) at url https://docs.accordproject.org/docs/started-hello.html, fixed in 42382175087d909d2c968bc52dad18c6a8810a9b
3. **remove additional space in code block** affecting 2 files (docs/started-hello.md, website/versioned_docs/version-0.21/started-hello.md) at url https://docs.accordproject.org/docs/started-hello.html, fixed in e8407f2323198c8030ac4c1d2aba67b78f0d80ef
4. **punctuation error("Note that for each template you can find samples for the text, for" .. -> "Note that for each template, you can find samples for the text, for" ..)** affecting 5 files (docs/started-hello.md, website/versioned_docs/version-0.12/basic-use.md ,website/versioned_docs/version-0.13/basic-use.md, website/versioned_docs/version-0.20/started-hello.md, website/versioned_docs/version-0.21/started-hello.md) at url https://docs.accordproject.org/docs/started-hello.html, fixed in b873e31c4b19ca431df352d1e58177d83b95defc
5. **grammatical error -  ("markdown rendering editing-") -> ("markdown rendering and editing")** affecting 2 files (docs/started-resources.md, website/versioned_docs/version-0.20/started-resources.md) at url https://docs.accordproject.org/docs/started-resources.html, fixed in 105dd109924736bb326cdc95aa33b165de4aafd2
6. **grammatical error -  add an "and" in "visualizing, creating, editing Accord Project templates", 
("visualizing, creating, editing Accord Project templates") -> ("visualizing, creating and editing Accord Project templates")** affecting 2 files (docs/started-resources.md, website/versioned_docs/version-0.20/started-resources.md) at url https://docs.accordproject.org/docs/started-resources.html, fixed in 105dd109924736bb326cdc95aa33b165de4aafd2
7. **grammatical error - ("Optional blocks enables text which depends on the presence of absence of an `optional` variable in your model")->("Optional blocks enable text which depends on the presence or absence of an `optional` variable in your model")** affecting 2 files (docs/markup-templatemark.md, website/versioned_docs/version-0.21/markup-templatemark.md) at url  https://docs.accordproject.org/docs/markup-templatemark.html, fixed in aac3e89fcef5b0bf47e43b61675b5517838eef2e
8. **grammatical error - ("A `with` block can be used to changes variables that are in scope in a specific part of a template grammar:")->("A `with` block can be used to change variables that are in scope in a specific part of a template grammar:")** affecting 3 files (docs/markup-templatemark.md, website/versioned_docs/version-0.20/markup-blocks.md, website/versioned_docs/version-0.21/markup-templatemark.md) at url  https://docs.accordproject.org/docs/markup-templatemark.html, fixed in aac3e89fcef5b0bf47e43b61675b5517838eef2e
9. **grammatical error - ("For instance, the following scenario describe the expected response for a given request to the [helloworld template](https://templates.accordproject.org/helloworld@0.10.1.html):")->("For instance, the following scenario describes the expected response for a given request to the [helloworld template](https://templates.accordproject.org/helloworld@0.10.1.html):")** affecting 4 files (docs/ref-cicero-testing.md, website/versioned_docs/version-0.12/ref-testing.md, website/versioned_docs/version-0.20/ref-testing.md, website/versioned_docs/version-0.21/ref-cicero-testing.md) at url  https://docs.accordproject.org/docs/ref-cicero-testing.html, fixed in 9b2fe2b21b46ba999e20b48c23f17b9a6e2b540d
10. **grammatical error - ("While the syntax for each scenario uses _pseudo_ natural language (e.g., `When it receives the request`), the tests much use very specific sentences as illustrated in this guide.")->("While the syntax for each scenario uses _pseudo_ natural language (e.g., `When it receives the request`), the tests use very specific sentences as illustrated in this guide.")** affecting 4 files (docs/ref-cicero-testing.md, website/versioned_docs/version-0.12/ref-testing.md, website/versioned_docs/version-0.20/ref-testing.md, website/versioned_docs/version-0.21/ref-cicero-testing.md) at url  https://docs.accordproject.org/docs/ref-cicero-testing.html, fixed in 9b2fe2b21b46ba999e20b48c23f17b9a6e2b540d
11. **grammatical error - ("For instance, the following scenario describe the expected response for the default contract text when sending the default request to the [helloworld template](https://templates.accordproject.org/helloworld@0.10.1.html):")->("For instance, the following scenario describes the expected response for the default contract text when sending the default request to the [helloworld template](https://templates.accordproject.org/helloworld@0.10.1.html):")** affecting 4 files (docs/ref-cicero-testing.md, website/versioned_docs/version-0.12/ref-testing.md, website/versioned_docs/version-0.20/ref-testing.md, website/versioned_docs/version-0.21/ref-cicero-testing.md) at url  https://docs.accordproject.org/docs/ref-cicero-testing.html, fixed in 9b2fe2b21b46ba999e20b48c23f17b9a6e2b540d
12. **grammatical error - ("For instance, the following scenario describe an expected error for a given request to the [Interest Rate Swap](https://templates.accordproject.org/interest-rate-swap@0.4.1.html) template:")->("For instance, the following scenario describes an expected error for a given request to the [Interest Rate Swap](https://templates.accordproject.org/interest-rate-swap@0.4.1.html) template:")** affecting 4 files (docs/ref-cicero-testing.md, website/versioned_docs/version-0.12/ref-testing.md, website/versioned_docs/version-0.20/ref-testing.md, website/versioned_docs/version-0.21/ref-cicero-testing.md) at url  https://docs.accordproject.org/docs/ref-cicero-testing.html, fixed in 9b2fe2b21b46ba999e20b48c23f17b9a6e2b540d
13. **grammatical error - ("For instance, the following scenario for the [Installment Sale](https://templates.accordproject.org/installment-sale@0.12.1.html) template describe the expected initial state and execution of one installment:")->("For instance, the following scenario for the [Installment Sale](https://templates.accordproject.org/installment-sale@0.12.1.html) template describe the expected initial state and execution of one installment:")** affecting 4 files (docs/ref-cicero-testing.md, website/versioned_docs/version-0.12/ref-testing.md, website/versioned_docs/version-0.20/ref-testing.md, website/versioned_docs/version-0.21/ref-cicero-testing.md) at url  https://docs.accordproject.org/docs/ref-cicero-testing.html, fixed in 9b2fe2b21b46ba999e20b48c23f17b9a6e2b540d
